### ISSUE TITLE
Do YAML comments need a space after the #?

### ISSUE THREAD

**Issue by prtksxna** [JujuAdams/SNAP#28]
This is working:

```yaml
videoediting:
  - l1
  - l2
# putOnTop:
#  - Door: ZuCoffee
```

Data I am getting: `{ videoediting : [ "l1","l2" ] }`

---
This is not working:

```yaml
videoediting:
  - l1
  - l2
#putOnTop:
#  - Door: ZuCoffee
```

Data I am getting: `{ videoediting : [ "l1","l2" ], #putOnTop : undefined }`

---

Do YAML comments need to have a space after `#`?

**Comment by JujuAdams** [comment]
All the examples I've seen use a space, but the spec appears to require that comments are only preceeded by whitespace. I will need to poke at a reference implementation.

**Comment by JujuAdams** [comment]
k, checked a few other implementations and it seems # varies a bit 🙄 I'll implement the requested change when I can as it's the more permissive option.

**Comment by prtksxna** [comment]
Fixed in 70cd9e141355c87fea3e17e0919b3ece1fd11090 and available in https://github.com/JujuAdams/SNAP/releases/tag/5.1.1

Thank you!
